cypress/e2e/side-panel.cy.js

```diff
@@ -18,4 +20,13 @@ describe(`Right side panel`, () => {
     cy.get('button[aria-label="Open Panel"]').click()
     cy.get('[data-testid="right-panel"]').should('be.visible')
   })
+
+  it('Should allow subscribing to the newsletter', () => {
+    cy.get('input[placeholder="Enter your email"]').type('kero@meilisearch.com')
```

Comment:
Will this trigger the subscription, or is it caught somewhere to not send it in testing environment?

Reply:
This triggers a subscription, but nothing happens when you subscribe with an existing email.

Reply:
Can you explain what you mean by "nothing happens when you subscribe with an existing email"?

Reply:
You can see that there are multiple submissions, but the contact is simply "upserted" there is no duplicate contacts created in HubSpot. You can see the forms submissions here: https://app-eu1.hubspot.com/submissions/25945010/form/991e2a09-77c2-4428-9242-ebf26bfc6c64/submissions?redirectUrl=https://app-eu1.hubspot.com/forms/25945010/views/all_forms

Reply:
Ok I'm wondering if we could have improved it by either providing a fake email or having intercepted the request to avoid spamming Hubspot's API 🤔 
Approving for now since it's urgent but we should consider doing this test in a different way 🥲